package.json

```
@@ -88,23 +110,26 @@
   ],
   "sideEffects": false,
   "dependencies": {
-    "@mswjs/cookies": "^0.2.2",
-    "@mswjs/interceptors": "^0.17.10",
-    "@open-draft/until": "^1.0.3",
+    "@bundled-es-modules/cookie": "^2.0.0",
+    "@bundled-es-modules/js-levenshtein": "^2.0.1",
+    "@bundled-es-modules/statuses": "^1.0.1",
+    "@mswjs/cookies": "^1.0.0",
+    "@mswjs/interceptors": "^0.25.1",
+    "@open-draft/until": "^2.1.0",
     "@types/cookie": "^0.4.1",
     "@types/js-levenshtein": "^1.1.1",
-    "chalk": "^4.1.1",
+    "@types/statuses": "^2.0.1",
+    "chalk": "^4.1.2",
     "chokidar": "^3.4.2",
-    "cookie": "^0.4.2",
+    "formdata-node": "4.4.1",
     "graphql": "^16.8.1",
-    "headers-polyfill": "3.2.5",
-    "inquirer": "^8.2.0",
+    "headers-polyfill": "^4.0.1",
```

```suggestion
    "headers-polyfill": "^4.0.1",
    "inquirer": "^8.2.0",
```

I believe this was removed unintentionally

---

@mattcosta7, thanks for spotting this! Looks like a faulty rebase over here. Do you have a moment to issue a pull request to this branch to fix this? 🙏

---

done - https://github.com/mswjs/msw/pull/1766

---

Thanks, @mattcosta7! Will release a new candidate later today.